JankStatsSample/app/build.gradle

```diff
@@ -41,15 +41,14 @@ android {
 
 dependencies {
     implementation "androidx.appcompat:appcompat:$appcompat_version"
-    implementation "androidx.core:core-ktx:$core_version"
+    implementation "androidx.compose.material:material:$compose_version"
```

Comment:
We had to have 2 versions in the Macrobench sample. Should we have them here as well?

Reply:
It works without, so I'd just keep it this way.